docs/release-notes.rst

```diff
@@ -6,6 +6,364 @@
+
+-  Experiment Config: the ``entrypoint`` field no longer accepts ``model_def:TrialClass`` as trial
+   definitions. Please invoke your training script directly (``python3 train.py``).
+
+-  Core API: the ``SearcherContext`` (``core.searcher``) has been deprecated. Training code no
+   longer requires ``core.searcher.operations`` to run, and progress should be reported through
+   ``core.train.report_progress``.
+
+-  DeepSpeed: the ``num_micro_batches_per_slot`` and ``train_micro_batch_size_per_gpu`` attributes
+   on ``DeepSpeedContext`` have been replaced with ``get_train_micro_batch_size_per_gpu()`` and
+   ``get_num_micro_batches_per_slot()``.
+
+-  Horovod: the horovod distributed training backend has been deprecated. Users are encouraged to
+   migrate to the native distributed backend of their training framework (``torch.distributed`` or
+   ``tf.distribute``).
+
+-  Trial APIs: ``TFKerasTrial`` has been deprecated. Users are encouraged to migrate to the new
+   :ref:`Keras Callback <api-keras-ug>`.
+
+-  Launchers: the ``--trial`` argument in Determined launchers has been deprecated. Please invoke
+   your training script directly.
+
+-  ASHA: the ``stop_once`` field of the ``searcher`` config for ASHA searchers has been deprecated.
+   All ASHA searches are now early-stopping based (``stop_once: true``) instead of promotion based.
+
+-  CLI: The ``--test`` and ``--local`` flags for ``det experiment create`` have been deprecated. All
+   training APIs now support local execution (``python3 train.py``). Please see ``training apis``
+   for details specific to your framework.
+
+-  Web UI: previously, trials that reported an ``epoch`` metric enabled an epoch X-axis in the Web
+   UI metrics tab. This metric name has been changed to ``epochs``, with ``epoch`` as a fallback
+   option.
+
+-  Cluster: A reminder that Amazon Aurora V1 will reach End of Life at the end of 2024. It is no
+   longer supported as the default persistent storage for AWS Determined deployments. We recommend
+   that users migrate to Amazon RDS for PostgreSQL. For more information, visit the `migration
+   instructions <https://gist.github.com/maxrussell/c67f4f7d586d55c4eb2658cc2dd1c290>`_.
+
+-  Cluster: After Amazon Aurora V1 reaches End of Life, support for Amazon Aurora V1 in ``det deploy
+   aws`` will be removed. Future deployments will default to the ``simple-rds`` type, which uses
+   Amazon RDS for PostgreSQL. Changes to the deployment code will ensure this transition to the new
+   default.
+
+-  Database: As a follow-up to the earlier notice, PostgreSQL 12 will reach End of Life on November
+   14, 2024. Instances still using PostgreSQL 12 or earlier should upgrade to PostgreSQL 13 or later
+   to maintain compatibility. The application will log a warning if it detects a connection to any
+   PostgreSQL version older than 12, and this warning will be updated to include PostgreSQL 12 once
+   it is End of Life.
+
+**************
```

Comment:
Looks like we've got some duplication? Not referring to the heading - I see that was duplicated the same way previously - but the whole list.

Reply:
yeah, i killed and reran rph. fixed now.